docker/docker-compose.ci.yml

```diff
@@ -9,6 +9,7 @@ services:
     environment:
       PORT: 8888
       SIRIUS_URL: http://json-server:3000
+      PREFIX: /supervision/deputies/public-authority
```

Comment:
This CI file should only be used in CI, and CI doesn't need the prefix (as it is only running tests). What's the reason for the change?

Reply:
Hm the cypress tests are failing now without the prefix in the build (it's coming back with a 404). I'm not sure why my change to the routing would've broken it as i have amended the url in the test file. If you look at a previous failing build you will see it's routing to the correct url so i suspect it's an issue with locating the asset files?

Reply:
That's because you updated the Cypress tests to use the prefix.

The reason the prefix wasn't given is just because it wasn't needed. The Cypress tests don't need to go via Sirius so there's no problem with them just addressing localhost. The user can't do this in production so there needs to be a prefix to route between the different Sirius platforms.

There's nothing wrong with adding in the prefix to the CI file. It just makes any call to localhost in the tests longer as you have to append the prefix.

Reply:
Ah ok thanks, it's also made a bit complicated by the fact that the `docker-compose.dev.yml ` file uses the prefix but the ci one doesn't so if i run the cypress tests against the dev file as i have it set up in the read me then the tests would fail if i remove the prefix from the cypress test file 

I'm guessing cypress tests should be ran against the ci file in that case and i should amend the readme?

`docker-compose -f docker/docker-compose.ci.yml up -d --build `
 
`yarn && yarn cypress `

Reply:
Ah okay. Yes, you could do that. However, that would mean you would need to stop your local dev environment in order to run your tests, which is not a great feedback loop.

Probably best then to leave the prefix in the ci file and keep them both the same.